PendingReleaseNotes

```diff
@@ -13,9 +13,12 @@
 * RBD: New `rbd_aio_compare_and_writev` API method to support scatter/gather
   on both compare and write buffers.  This compliments existing `rbd_aio_readv`
   and `rbd_aio_writev` methods.
+* RBD: `rbd device unmap` command gained `--namespace` option.  Support for
+  namespaces was added to RBD in Nautilus 14.2.0 and it has been possible to
+  map and unmap images in namespaces using the `image-spec` syntax since then
+  but the corresponding option available in most other commands was missing.
 
 >=17.2.4
---------
```

Comment:
This line (removal of `--------`) is the only thing that flags this as a non-clean backport.  I'd get rid of it.

Reply:
It was rebasing after #48474 merge. Should be fine now

Reply:
<img width="916" alt="image" src="https://user-images.githubusercontent.com/7759548/196147187-623695c4-1ca2-4bd6-af87-ef083df711bf.png">

Reply:
Browser cache?

Reply:
Now I see... 🙂
Repushed 👈

Reply:
Thanks, LGTM!